Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarCell.kt

```diff
@@ -34,7 +34,7 @@ internal sealed class CalendarCell {
   abstract val id: Long
 }
 
-internal data class CalendarSpace(
+internal data class CalendarCellSpace(
```

Comment:
I like to put data classes for sealed classes within the sealed class and then you get the `CalendarCell` name from the sealed class and can name the data class just `Space` - e.g. you'd refer to it as `CalendarCell.Space`

Reply:
In was done in this manner, but I moved in out in order not to have too much code in the parent classes. What do you think, shall we put it back?

Reply:
I generally prefer it as the context is clearer and if you rename `CalendarCell` you only have to rename it once, rather than in every implementation